docs/content.zh/docs/connectors/table/filesystem.md

```diff
@@ -190,209 +194,217 @@ CREATE TABLE MyUserTableWithFilepath (
         <td>String</td>
-        <td>The time zone to parse the long watermark value to TIMESTAMP value, the parsed watermark timestamp is used to compare with partition time to decide the partition should commit or not. This option is only take effect when `sink.partition-commit.trigger` is set to 'partition-time'. If this option is not configured correctly, e.g. source rowtime is defined on TIMESTAMP_LTZ column, but this config is not configured, then users may see the partition committed after a few hours. The default value is 'UTC', which means the watermark is defined on TIMESTAMP column or not defined. If the watermark is defined on TIMESTAMP_LTZ column, the time zone of watermark is the session time zone. The option value is either a full name such as 'America/Los_Angeles', or a custom timezone id such as 'GMT-08:00'.</td>
+        <td> 解析 Long 类型的 watermark 到 TIMESTAMP 类型时所采用的时区，解析得到的 watermark 的 TIMESTAMP 会被用来跟分区时间进行比较以判断是否该被提交。这个属性仅当 `sink.partition-commit.trigger` 被设置为 'partition-time' 时有效。如果这个属性设置的不正确，例如，在 TIMESTAMP_LTZ 类型的列上定义了 source rowtime，如果没有设置该属性，那么用户可能会在若干个小时后才看到分区的提交。默认值为 'UTC'，意味着 watermark 是定义在 TIMESTAMP 类型的列上或者没有定义 watermark。如果 watermark 定义在 TIMESTAMP_LTZ 类型的列上，watermark 时区必须是会话时区（session time zone）。该属性的可选值要么是完整的时区名比如 'America/Los_Angeles'，要么是自定义时区，例如 'GMT-08:00'。</td>
     </tr>    
   </tbody>
 </table>
 
-There are two types of trigger:
-- The first is partition processing time. It neither requires partition time extraction nor watermark
-  generation. The trigger of partition commit according to partition creation time and current system time. This trigger
-  is more universal, but not so precise. For example, data delay or failover will lead to premature partition commit.
-- The second is the trigger of partition commit according to the time that extracted from partition values and watermark.
-  This requires that your job has watermark generation, and the partition is divided according to time, such as
-  hourly partition or daily partition.
+两种类型分区提交触发器：
+- 第一种是根据分区的处理时间。既不需要额外的分区时间，也不需要 watermark 生成。这种分区提交触发器基于分区创建时间和当前系统时间。
+  这种触发器更具普遍性，但不是很精确。例如，数据延迟或故障将导致过早提交分区。
+- 第二种是根据从分区字段提取的时间以及 watermark。
+  这需要 job 支持 watermark 生成，分区是根据时间来切割的，例如，按小时或按天分区。
 
-If you want to let downstream see the partition as soon as possible, no matter whether its data is complete or not:
-- 'sink.partition-commit.trigger'='process-time' (Default value)
-- 'sink.partition-commit.delay'='0s' (Default value)
-  Once there is data in the partition, it will immediately commit. Note: the partition may be committed multiple times.
+如果想让下游尽快感知到分区，不管分区数据是否完整：
+- 'sink.partition-commit.trigger'='process-time' (默认值)
+- 'sink.partition-commit.delay'='0s' (默认值)
+  一旦数据进入分区，将立即提交分区。注意：这个分区会被提交多次。
 
-If you want to let downstream see the partition only when its data is complete, and your job has watermark generation, and you can extract the time from partition values:
+如果想让下游只有在分区数据完整时才感知到分区，并且 job 中有 watermark 生成，也能从分区字段的值中提取到时间：
```

Comment:
```suggestion
如果你想让下游只有在分区数据完整时才感知到分区，并且 job 中有 watermark 生成，也能从分区字段的值中提取到时间：
```

Reply:
Sorry, I don't think it's necessary to add 'you' in Chinese!